master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/policy-only/1.7/calico.yaml

```diff
@@ -243,9 +243,6 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            # No IP address needed.
```

Comment:
Maybe we should use `autodetect` here?  If we don't then we won't benefit from the fix in #1589 when IPs change (since we deliberately exclude the `IP=""` case in that patch).  Looks like, if `IP` is not set then we do a single autodetection anyway?

Reply:
Hm, this is the policy-only manifest, which shouldn't care about the BGP IP address anyway. 

The code in #1589 won't even get hit because we set `CALICO_NETWORKING_BACKEND=none` in this manifest.

Reply:
Ah OK, that makes sense, was a bit tunnel-visioned on the code I was reviewing.